src/components/simple-color-picker/simple-color-picker.component.tsx

```diff
@@ -122,19 +181,25 @@ const SimpleColorPicker = (props) => {
 
       e.preventDefault();
 
-      let itemIndex;
+      let itemIndex: number | string | null = null;
```

Comment:
`target.getAttribute("data-item-down")` returns a string
`navigationGrid.findIndex(position)` returns number

And I have to initialize it with `null` value otherwise I get this error: `Variable 'itemIndex' is used before being assigned.ts(2454)`

Reply:
we should avoid having this kind of multi-type variables in TypeScript.
The variable should be a `number` and the result of the `getAttribute` should be converted to that type.
Possibility of that being null shows us that there is something wrong with the code.